torch/jit/_recursive.py

```diff
@@ -53,7 +53,9 @@ def copy_to_script_module(original, stubs):
             if (name in original._parameters or name in original._buffers) and item is not None:
                 # for 'None' parameters/buffers, don't actually add their values if it exists
                 continue
-            setattr(script_module, name, getattr(original, name))
+            # don't recopy constantss
```

Comment:
This should only happen for constant modules / params, could you note that in the comment here?